js/editor-cm.js

```diff
@@ -1,5 +1,108 @@
 var EditSession = CodeMirror.Doc;
 
+// taken from a chrome dev tools
+// go/codemirror-a11y-hack
+CodeMirror.inputStyles.devToolsAccessibleTextArea = class extends CodeMirror.inputStyles.textarea {
+  /**
+   * @override
+   * @param {!Object} display
+   */
+  init(display) {
+    super.init(display);
+    this.textarea.addEventListener('compositionstart', this._onCompositionStart.bind(this));
+
+  }
+
+  _onCompositionStart() {
```

Comment:
better to make the function name describe what it does rather than when it's called (other places in text app are guilty of this but let's not add to it)

Reply:
Code was killed in this PR